dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/domain/Account.java

```diff
@@ -44,36 +44,36 @@
    @Nullable
    public abstract String customDefined2();
 
-   public abstract String orgId();
+   @Nullable
+   public abstract String state();
 
    @Nullable
-   public abstract Roles roles();
+   public abstract List<RoleType> roles();
 
-   Account() {
+   @Nullable
+   public abstract AccountPhoneNumber phone();
+
+   public abstract AccountOrganization organization();
+
+   @SerializedNames({ "userName", "fullName", "firstName", "lastName", "emailAddress", "roles", "phone", "department",
+         "customDefined1", "customDefined2", "organization", "state" })
+   public static Account create(String userName, String fullName, String firstName, String lastName,
+         String emailAddress, List<RoleType> roles, AccountPhoneNumber phone, String department, String customDefined1,
+         String customDefined2, AccountOrganization organization, String state) {
+      return builder().userName(userName).fullName(fullName).firstName(firstName).lastName(lastName)
+            .emailAddress(emailAddress)
+            .roles(roles == null ? ImmutableList.<RoleType>of() : ImmutableList.copyOf(roles)).phone(phone)
```

Comment:
Just pass the list here as it comes from the parameter, and move the immutability logic to the builder following [this pattern](https://github.com/jclouds/jclouds-labs/blob/master/dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/domain/Backup.java#L53-L62).